lib/cosmos/packets/packet.rb

```diff
@@ -454,6 +454,11 @@ def get_item(name)
     #   can be any type.
     def read_item(item, value_type = :CONVERTED, buffer = @buffer)
       value = super(item, :RAW, buffer)
+      derived_raw = false
+      if item.data_type == :DERIVED && value_type == :RAW
+        value_type = :CONVERTED
+        derived_raw = true
+      end
```

Comment:
Cool.  Now DERIVED has RAW that has a meaning (basically converted but not states)